src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" 
+        alt="Apple Retina Monoblock" 
+        width="160px" height="134px" 
+        class="card__image"
+      >
+  
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <span class="card__product-code">
+        Product code: 195434
+      </span>
+
+      <div class="card__opinion">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--non-active"></div>
+        </div>
+    
+        <div class="stars_reviews">
+          Reviews: 5
+        </div>
+      </div>
+  
+
+      <div class="card__price">
+        <span class="card__price-text">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </div>
+
+      <a href="/#" class="card__buy-link" 
+      data-qa="hover"
+      >
+        buy
+      </a>
+
+    </div>
```

Comment:
Don't add empty lines between parents and child components
```suggestion
      </a>
    </div>
```